test/extended/testdata/build-secrets/test-docker-build.json

```diff
@@ -25,17 +25,17 @@
     },
     "strategy":{
       "type":"Docker",
-      "env": [
-        {
-          "name": "BUILD_LOGLEVEL",
-          "value": "5"
-        }
-      ],
```

Comment:
was this just always in the wrong place?  if so, how are we testing this log level setting functionality?